### PR TITLE
remove empty media queries

### PR DESCRIPTION
**Pull request by HellPat**
Fixes issue from that comment: https://github.com/tailwindcss/tailwindcss/issues/16#issuecomment-341097747

@adamwathan wants this to happen, so here it is ;-)

https://github.com/tailwindcss/tailwindcss/issues/16#issuecomment-341097747

Empty media queries are added if i do not use ```@tailwind utilities``` and use only preflight.

### CODE REVIEW

File: src/lib/substituteResponsiveAtRules.js
Comment (adamwathan) [inline src/lib/substituteResponsiveAtRules.js]: Minor style changes but can you add a space after the `if`, and remove the comment? I think the code is expressive enough to me without it 👍

File: __tests__/sanity.test.js
Comment (adamwathan) [inline __tests__/sanity.test.js]: Another minor change but can you make sure this file ends with a newline character?

**Review by adamwathan — CHANGES_REQUESTED** [review]
Awesome, thanks @psren! A couple minor tweaks but happy to merge right after that.

File: __tests__/sanity.test.js
Comment (adamwathan) [inline __tests__/sanity.test.js]: One more comment actually! Try to name the tests so that they read properly starting with the word "it". In this case maybe this would be a good name:

"does not add any CSS if no Tailwind features are used"

File: __tests__/sanity.test.js
Comment (psren) [reply __tests__/sanity.test.js]: done

File: src/lib/substituteResponsiveAtRules.js
Comment (psren) [reply src/lib/substituteResponsiveAtRules.js]: done

File: __tests__/sanity.test.js
Comment (psren) [reply __tests__/sanity.test.js]: done and thanks for the input :-)

**Review by adamwathan — APPROVED** [review]

**Comment by adamwathan** [comment]
Awesome, thanks @psren!

**Pull request merged**
